app/app.js

```diff
@@ -24,6 +24,12 @@ const Menu = remote.Menu;
 const MenuItem = remote.MenuItem;
 
 const tinycolor = require('tinycolor2');
+const nunjucks = require('nunjucks');
```

Comment:
Sorry, didn't get your comment about Multiple environments.

Second example can be acheived with:
format: "$material-color('$HUE', '$VALUE');"
transform: "-x"?